apps/ideas/templates/advocate_europe_ideas/idea_detail.html

```diff
@@ -9,14 +9,25 @@
     <meta name="twitter:card" content="summary_large_image">
     <meta name="twitter:title" content="Advocate Europe | {{ idea.idea_title }}">
     <meta name="twitter:description" content="{{ idea.idea_pitch }}">
-    <meta name="twitter:image" content="{% get_absolute_uri obj=view.idea_image.url %}">
+    {% if idea.idea_image_exists %}
+        <meta name="twitter:image" content="{% get_absolute_uri obj=idea.idea_image.url %}">
+    {% else %}
+        <meta name="twitter:image" content="{% get_absolute_uri_static obj='images/logo_01.png' %}">
+    {% endif %}
+
 
     <meta property="og:url" content="{{ request.build_absolute_uri }}">
     <meta property="og:title" content="Advocate Europe | {{ idea.idea_title }}">
     <meta property="og:description" content="{{ idea.idea_pitch }}">
-    <meta property="og:image" content="{% get_absolute_uri obj=view.idea_image.url %}">
-    <meta property="og:image:width" content="{{ view.idea_image.width }}">
-    <meta property="og:image:height" content="{{ view.idea_image.height }}">
+    {% if idea.idea_image_exists %}
+        <meta property="og:image" content="{% get_absolute_uri obj=idea.idea_image.url %}">
+        <meta property="og:image:width" content="{{ idea.idea_image.width }}">
+        <meta property="og:image:height" content="{{ idea.idea_image.height }}">
+    {% else %}
+        <meta property="og:image" content="{% get_absolute_uri_static obj='images/logo_01.png' %}">
+        <meta property="og:image:width" content="360">
```

Comment:
No one will ever change this if a new version of the logo is uploaded or similar.